public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -463,10 +425,6 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
       params.db = this.database;
     }
 
-    if (options?.policy) {
```

Comment:
why does this need to be removed? 

wont this break this functionality from this pr https://github.com/grafana/grafana/pull/62149/files

Reply:
This was a broken functionality anyways. 
> we try to query with default retention policy and if there is no such policy, the query fails.
https://github.com/grafana/grafana/pull/63820

Reply:
Since we are adding the retention policy to the query string I think this is unnecessary. But if my claim is wrong I can put it back and just not add `default` retention policy to the request.

Reply:
i think it was needed for this edge case:

In InfluxDB v2, due to technical limitations of the InfluxDB v1 compatibility layer, retention policies in a query (e.g. <retention policy>.<measurement_name>) aren't honored and must be specified in the URL query parameter rp to be applied

Reply:
https://github.com/grafana/grafana/pull/72467/commits/0a7d8c44be60fbd708c90761d96c9e41f08f7500